### ISSUE TITLE
TypeError: '<=' not supported between instances of 'NoneType' and 'float'

### ISSUE THREAD

**Issue by jtojnar** [pytrainer/pytrainer#172]
If I open the :heart: tab under the Record tab for an entry without a heart rate data, I get the following error:

```
Traceback (most recent call last):
  File "/home/jtojnar/projects/pytrainer/pytrainer/gui/windowmain.py", line 1678, in on_recordpage_change
    self.parent.refreshRecordGraphView(selected_view)
  File "/home/jtojnar/projects/pytrainer/pytrainer/main.py", line 332, in refreshRecordGraphView
    self.windowmain.actualize_heartrategraph(activity)
  File "/home/jtojnar/projects/pytrainer/pytrainer/gui/windowmain.py", line 773, in actualize_heartrategraph
    self.drawareaheartrate.drawgraph(activity.tracks)
  File "/home/jtojnar/projects/pytrainer/pytrainer/heartrategraph.py", line 35, in drawgraph
    drawarea.stadistics(shape,[xvalues],[yvalues],[xlabel],[ylabel],[title],[color],zones)
  File "/home/jtojnar/projects/pytrainer/pytrainer/gui/drawArea.py", line 53, in stadistics
    self.drawZones(type,xvalues,yvalues,xlabel,ylabel,title,color,zones)
  File "/home/jtojnar/projects/pytrainer/pytrainer/gui/drawArea.py", line 409, in drawZones
    if value <= zones[2][1]:
TypeError: '<=' not supported between instances of 'NoneType' and 'float'

```

**Comment by avtobiff** [comment]
I can't reproduce this behaviour. Can you supply an example config
setup where this occurs?

**Comment by jtojnar** [comment]
While I can no longer reproduce the issue with the above steps, the following still produce the error.

1. Delete `~/.pytrainer` directory.
2. Start PyTrainer
3. Click on _File_ » _New_.
4. Fill in a title.
5. Open a GPX file. I can provide the GPX file in private if necessary.
6. Press _Calculate values_.
7. Press _OK_
8. Choose _View_ » _List_ view.
9. Right click the record and click _Show record_ in _classic_ view
10. Open the :heart: tab under the _Record_ tab

This is on Pytrainer 2.0.0 in a clean `~/.pytrainer` directory, the following error will appear in the console:

```
Traceback (most recent call last):
  File "/nix/store/vrvbpzyyqg2p8zk2c56dykj7rsc93039-pytrainer-2.0.0/lib/python3.7/site-packages/pytrainer/gui/windowmain.py", line 1686, in on_recordpage_change
    self.parent.refreshRecordGraphView(selected_view)
  File "/nix/store/vrvbpzyyqg2p8zk2c56dykj7rsc93039-pytrainer-2.0.0/lib/python3.7/site-packages/pytrainer/main.py", line 331, in refreshRecordGraphView
    self.windowmain.actualize_heartrategraph(activity)
  File "/nix/store/vrvbpzyyqg2p8zk2c56dykj7rsc93039-pytrainer-2.0.0/lib/python3.7/site-packages/pytrainer/gui/windowmain.py", line 773, in actualize_heartrategraph
    self.drawareaheartrate.drawgraph(activity.tracks)
  File "/nix/store/vrvbpzyyqg2p8zk2c56dykj7rsc93039-pytrainer-2.0.0/lib/python3.7/site-packages/pytrainer/heartrategraph.py", line 35, in drawgraph
    drawarea.stadistics(shape,[xvalues],[yvalues],[xlabel],[ylabel],[title],[color],zones)
  File "/nix/store/vrvbpzyyqg2p8zk2c56dykj7rsc93039-pytrainer-2.0.0/lib/python3.7/site-packages/pytrainer/gui/drawArea.py", line 53, in stadistics
    self.drawZones(type,xvalues,yvalues,xlabel,ylabel,title,color,zones)
  File "/nix/store/vrvbpzyyqg2p8zk2c56dykj7rsc93039-pytrainer-2.0.0/lib/python3.7/site-packages/pytrainer/gui/drawArea.py", line 409, in drawZones
    if value <= zones[2][1]:
TypeError: '<=' not supported between instances of 'NoneType' and 'float'
```

**Comment by hjerning** [comment]
This should be fixed by 52c763ae.
